### DISCUSSION TITLE
Rename the settings page?

### DISCUSSION THREAD

**Discussion by jeffalo** [ScratchAddons/ScratchAddons#114] [Discussion]
since the settings page is more or less a dashboard instead of a settings page, perhaps it should be referred to as a dashboard instead of settings?
![image](https://user-images.githubusercontent.com/40470736/91399050-8f1e5400-e83d-11ea-9a37-b58070706f29.png)

but i guess it technically is a settings page... it just seems a bit wrong for it to be called a settings page in my opinion

**Comment by Nagol12344** [comment]
that's a good idea

**Comment by easrng** [comment]
Should we make clicking the icon open it?

**Comment by WorldLanguages** [comment]
> Should we make clicking the icon open it?

No, the popup will open when doing that.
However, the popup will have a clear way to open settings.
Keep in mind, the current plan is to make Scratch Addons replace the Scratch Messaging Extension, and all of its core functionality resides in the popup, so we must have a popup.

**Comment by easrng** [comment]
We should probably have a mini-dashboard for the pop-up to switch on and
off addons

**Comment by WorldLanguages** [comment]
Re: original post
I don't think this is a good idea. The fact there's no mention of "addons", "store", "dashboard", etc. is to try to avoid issues when the Chrome Web Store reviews the extension. They don't really like when a single extension offers lots of different functionality - this will be fine since they're all related to one thing (Scratch), but anyway, to avoid any issues, I believe it's a good idea to keep the settings page as close to a normal settings page, forgetting these are actually addons.

**Comment by WorldLanguages** [comment]
> We should probably have a mini-dashboard for the pop-up to switch on and off addons

Good idea. However I'll first work on the messaging addon popup, and we'll see later what else we can do with it.

**Comment by jeffalo** [comment]
interesting, ok.

**Comment by Explosion-Scratch** [comment]
How do I find this page?

**Comment by Explosion-Scratch** [comment]
I have the addon installed on chrome and it seems that it does nothing except show the number of messages I have... =/

**Comment by jeffalo** [comment]
@Explosion-Scratch 
![image](https://user-images.githubusercontent.com/40470736/91466310-39729780-e88f-11ea-936a-e57e78e653c7.png)

**Comment by WorldLanguages** [comment]
Yeah, as I said in another issue, there will be a way to access settings from the popup, to avoid exactly this.
